Add NPM publishing instructions to README (#120)
nucypher/nucypher-contracts: README.md

The networks used here are standard ape networks, you can see the full list with:
```
$ ape networks list
```

## NPM publishing process

> When a new smart contract is deployed it is needed to add to NPM package:

Reviewer: ```suggestion
For interoperability, we keep an NPM package with information of deployed smart contracts, such as address, ABI, etc.
```

> https://www.npmjs.com/package/@nucypher/nucypher-contracts

Reviewer: ```suggestion
The NPM package can be found in https://www.npmjs.com/package/@nucypher/nucypher-contracts and the process to update it is as follows:
```

> 1. Download the last version of the package in a separated folder. Ropsten versions end in
> `-ropsten`.

Reviewer: ```suggestion
1. Download the last version of the package in a separate folder. Testnet versions end in
`-<testnet>` (e.g., `-goerli`).
```
